src/js/view/view.js

```diff
@@ -53,6 +53,8 @@ define([
             _rightClickMenu,
             _resizeMediaTimeout = -1,
             _resizeContainerRequestId = -1,
+            _resizeRAFFunc = window.requestAnimationFrame || function(rafFunc) { return window.setTimeout(rafFunc, 17); },
```

Comment:
The name of the function is really hard to follow what that function is doing - can we rename this to something more intuitive or leave a comment on what resizeRAFFunc means? Same for ```_resizecCancelRAFFunc```.
Also, don't forget to rebase your commits :)